workflows/prognostic_c48_run/runtime/transformers/fv3fit.py

```diff
@@ -34,7 +34,8 @@ class Adapter:
     timestep: float
 
     def __post_init__(self: "Adapter"):
-        self.model = fv3fit.load(self.config.url)
+        models = [fv3fit.load(url) for url in self.config.url]
+        self.model = MultiModelAdapter(models)  # type: ignore
```

Comment:
`MultiModelAdapter` expects models that have been wrapped with `RenamingAdapter`, but I don't want to bother with that complication here. In practice no one ever uses the renaming feature anyway.